module/programUsers/helper.js

```diff
@@ -80,12 +80,14 @@ module.exports = class ProgramUsersHelper {
      * @param {Array} [filterData = "all"] - template filter query.
      * @param {Array} [fieldsArray = "all"] - projected fields.
      * @param {Array} [skipFields = "none"] - field not to include
+     * @param {Sort} [sort = false] - require sorting or not
      * @returns {Array} Lists of program user. 
     */
     static programUsersDocuments(
         filterData = "all", 
         fieldsArray = "all",
-        skipFields = "none"
+        skipFields = "none",
+        sort = false
```

Comment:
@VISHNUDAS-tunerlabs - Sorting param and direction should be passed by the calling function - Refer - https://github.com/shikshalokam/ml-core-service/blob/master/module/entities/helper.js